exercises/binary-search/example.h

```diff
@@ -0,0 +1,12 @@
+#if !defined(BINARY_SEARCH_H)
+#define BINARY_SEARCH_H
+
+#include <vector>
+
+namespace binary_search {
+
+int find(std::vector<int> const&, int);
```

Comment:
This should return a `size_t`, since it is returning an index.

Reply:
OK, changed.